cucumber/authenticators_gce/features/authn_gce_hosts.feature

```diff
@@ -135,4 +135,22 @@ Feature: GCP Authenticator - Test hosts can authentication scenarios
     And The following appears in the audit log after my savepoint:
     """
     cucumber:user:test-app successfully authenticated with authenticator authn-gce service cucumber:webservice:conjur/authn-gce
-    """
\ No newline at end of file
+    """
+
+  Scenario: Hosts defined outside of root can authenticate with GCE authenticator and fetch secret
+    Given I have host "non-rooted/test-app"
+    And I set all valid GCE annotations to host "non-rooted/test-app"
+    And I grant group "conjur/authn-gce/apps" to host "non-rooted/test-app"
+    And I have a "variable" resource called "test-variable"
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I permit host "non-rooted/test-app" to "execute" it
+    And I set all valid GCE annotations to host "test-app"
+    And I obtain a GCE identity token in full format with audience claim value: "conjur/cucumber/host/non-rooted/test-app"
+    And I save my place in the log file
+    When I authenticate with authn-gce using valid token and existing account
+    Then host "non-rooted/test-app" has been authorized by Conjur
```

Comment:
That sounds like 3 different tests
Would it be easy to understand what went wrong by looking at Jenkins output?

Reply:
what sounds like 3 different tests? what are the 3 tests?